common/execution_context.go

```diff
@@ -307,9 +309,7 @@ func (e *ExecutionContext) Eval(apiCtx context.Context, js string, args ...any)
 		returnByValue: true,
 	}
 	evalArgs := make([]any, 0, len(args))
-	for _, a := range args {
-		evalArgs = append(evalArgs, a)
-	}
+	evalArgs = append(evalArgs, args...)
```

Comment:
Do we need here (and same applies for the file) this `evalArgs` at all? :thinking:

Reply:
I have no idea - fixing nearly 400 lint errors and some point means that I stopped trying to figure out if a given piece of code is needed or I can remove it as goign up and down the stack is way too much work for each call place.

Reply:
Does look like we might not need it. I'll need to run some tests though. I think for this PR, it's probably best not to incase there's a good reason for it.